documentation/staging/1.4/content/release/_index.md

```diff
@@ -5,34 +5,29 @@ weight = 7
 pre = "<b> </b>"
 +++
 
-### Changes in Release 1.3.0
+### Changes in Release 1.4.0
 - [Major New Features](#major-new-features)
 - [Other Changes](#other-changes)
 - [Bugs Fixes](#bug-fixes)
 - [Known Issues](#known-issues)
 
 
 #### Major New Features
-- Support for creating and deploying the Verrazzano component for the WebLogic domain, and an application that references it.  
+- #173: Added support for WebLogic Kubernetes Operator 4.0.  While it is now the default, versions 3.3+ are still supported.  
 
 #### Other Changes
-- Updated the [WebLogic Remote Console](https://github.com/oracle/weblogic-remote-console) integration to support version 2.4.0.
-- Added support for specifying node selectors when installing the WebLogic Kubernetes Operator and deploying the WebLogic
-  domain, either directly or as a Verrazzano component.
-- Added support for the user specifying the Helm timeout value when helm is used (i.e., Installing/updating WebLogic
-  Kubernetes Operator and installing the Ingress Controller).
-- Added tooltips across all icon buttons to describe what they do.
-- Added an anti-affinity specification to the generated WebLogic domain resource, as recommended by the WebLogic Kubernetes Operator docs.
+- #175: Added Zoom In, Zoom Out, and Actual Size menu options to the Window menu.
+- #178: Integrating the Prepare Model action with the changes in WebLogic Deploy Tooling 2.4.0.
```

Comment:
Integrating -> Integrated (? to be parallel with other bulleted items)

(To consider, in Known Issues, we use code font for the `Check Application Status` button. If you like that, then we should use code font for `Prepare Model`, `Install Ingress Controller`, and `Update Ingress Routes`. If not, then let's not use it on the buttons/actions in the RNs.)